portfolio/src/main/webapp/script.js

```diff
@@ -14,15 +14,21 @@
 
 // gets the comments from the java servlet
 function getComments() {
+    // retrieve limit number of comments from the server
     var limit = document.getElementById("limit").value;
     fetch('/data?limit=' + limit).then(response => response.json()).then(commentsHistory => {
         const commentsEltList = document.getElementById('comments-container');
-        commentsEltList.innerHTML = '';
-        
-        commentsHistory.forEach(comment => {
-            commentsEltList.appendChild(createListElement(comment));
-        });
 
+        var countKey = Object.keys(commentsHistory).length;
```

Comment:
nit: Maybe rename to numComments?

Reply:
Sorry if there was any confusion, but I meant renaming countKey to numComments, since the number of keys should be the number of comments?